AtomLinter/linter-pylint
# Update: Use latest linter API

* Use latest atom linter API
* Use helpers from atom-linter
* Refactor to use latest linter plugin conventions (main.coffee, etc.)

#### Testing

This is a significant change, so it would be great if folks could test it out to ensure it works with their environment/config.

1. Checkout this PR ([instructions](https://gist.github.com/piscisaureus/3342247))
2. `cd` into your `linter-pylint` directory
3. Install into your dev packages - `apm link --dev`
4. Run atom in dev mode - `atom -d`

Using these directions will not affect your current linter-pylint.  You can go back to normal by running atom without the `-d` flag.

If you do run into issues, please comment and provide:

* OS and version
* atom version
* `linter` version
* Reasonably detailed description of the problem

Closes #52 #53 #51 #50 #49
